src/main.rs

```diff
@@ -148,6 +151,14 @@ async fn run() -> anyhow::Result<()> {
             ACCESS_CONTROL_ALLOW_ORIGIN,
             HeaderValue::from_static("*"),
         ))
+        .layer(GovernorLayer {
+            config: Arc::new(
+                GovernorConfigBuilder::default()
```

Comment:
Note: https://docs.rs/tower_governor/latest/tower_governor/governor/struct.GovernorConfig.html#method.default has

> The default configuration which is suitable for most services. Allows bursts with up to eight requests and replenishes one element after 500ms, based on peer IP.

I guess let's start with defaults but I feel that we might want to raise it for any sensible performance on zarrs.

Also eventually authenticated clients could get more generous limits, and platform seems to allow some customizations there.